app/controllers/ui_job_wizard_controller.rb

```diff
@@ -40,7 +40,7 @@ def resource_name(nested_resource = nil)
   end
 
   def with_katello
-    !!defined?(::Katello)
+    !!defined?(::Katello::Engine)
```

Comment:
Just a question, why would we check the constants if we could check it via plugin's API?

Reply:
None in particular, changed